<launch>
  
  <arg name="bag_path" default="" /> 
  <arg name="save_dir" default="" />
  <arg name="publish_framerate" default="60.0" />
  
  <!-- complementary filter -->
  <node name="reconstruction_node" pkg="complementary_filter" type="reconstruction_node" output="screen" args="--v=3">
    <param name="working_dir" value="$(env PWD)"/>
    <param name="bag_path" type="str" value="$(arg bag_path)"/>
    <param name="save_dir" type="str" value="$(arg save_dir)"/>
	<param name="publish_framerate" type="double" value="$(arg publish_framerate)"/>
	
    <param name="contrast_threshold_recalibration_frequency" type="double" value="10.0"/>
    
    <remap from="events" to="dvs/events" />
    <remap from="image_raw" to="dvs/image_raw" />
  </node>

</launch>

